.github/workflows/test-dbt-installation-docker.yml

```diff
@@ -119,15 +135,46 @@ jobs:
           image: ${{ steps.image-info.outputs.ref }}
           run: dbt --version
 
+      - name: "Dump Job Status"
+        if: ${{ always() }}
+        id: job-status
+        run: |
+          file="test-${{ strategy.job-index }}-tag-${{ matrix.tag }}.json"
+          # Create file
+          touch $file 
+          # Write job status to file
+          echo $JOB_STATUS >> $file
+          # Set path to file for subsequent steps          
+          echo "path=$file" >> $GITHUB_OUTPUT
+        env:
+          JOB_STATUS: ${{ toJson(job.status) }}
+
+      - name: "Upload Job Status"
```

Comment:
```suggestion
      - name: "Upload Job Status for Slack Notification"
```

Reply:
Fixed in 527e36accd5de2916c5d51b15b5f8ab7a6ccdd48